pywemo/subscribe.py

```diff
@@ -307,20 +311,16 @@ def _resubscribe(
     ) -> None:
         path = url_fn(device).rsplit('/')[-1]
         LOG.info("Resubscribe for %s %s", path, device)
-        headers = {'TIMEOUT': 'Second-300'}
-        if sid is not None:
-            headers['SID'] = sid
-        else:
-            host = get_ip_address(host=device.host)
-            headers.update(
-                {
-                    "CALLBACK": '<http://%s:%d/%s>' % (host, self.port, path),
-                    "NT": "upnp:event",
-                }
-            )
+        self.devices[device.host] = device  # If rediscovered with a new IP.
```

Comment:
This looks fine - although I think we turned off this sort of shape shifting!

Reply:
I added the comment to make this stand out. I didn't want to hide it. It worked! :)

As long as discovery is involved in reconnecting we'll still have some outlier cases where the IP changes and pyWeMo picks up those changes. I felt this change this was in line with the 80:20 law (or in this case 99.9:0.1 law) :) It doesn't really impact the 99.9% of cases, and closes one small gap in the 0.1% case.